ISISScientificComputing/autoreduce
# Python 3

### Summary of work
* All files moved to be python3 compatible
* Travis running python 3 
* Dev environment updated and tested

### How to test your work
* Ensure we can merge with master
* Ensure travis passes


Fixes #454
